crates/telio-proxy/src/proxy.rs

```diff
@@ -323,8 +334,18 @@ impl StateEgress {
                         return;
                     }
                 };
+
                 if let Some(pk) = pk {
-                    self.sockets.insert(pk, (Arc::new(socket), None));
+                    let socket = Arc::new(socket);
+
+                    if let Err(e) = self.update_socket.send((pk, socket.clone())).await {
+                        // May happen when they are being closed, but is not critical
+                        telio_log_debug!(
```

Comment:
Would it be possible to somehow make this `telio_log` to have verbosity level info or above, but in case of many errors avoid spamming? It feels like issue syncing up sockets between two tasks should be something, that raises our attention in the logs if it occurs.

Reply:
It's of course possible, but I wonder if the effort is really worth benefit compared to just doing the same level all the time. I can definitely change it to info or warn though